homeassistant/components/isy994/services.py

```diff
@@ -492,13 +499,18 @@ def async_log_deprecated_service_call(
         },
     )
 
+    alternate = (
+        f' and pass it a target entity ID of "{alternate_target}"'
+        if alternate_target
+        else ""
+    )
```

Comment:
Please unpack this into an `if` block since it spans multiple lines.

Reply:
Done